python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -1104,15 +1106,15 @@ async def move_m2_hexapod(
             Hexapod-v angle (degrees).
         w : `float`, optional
             Hexapod-w angle (degrees). Default 0.
-        sync : `bool`, optinal
+        sync : `bool`, optional
             Should the hexapod movement be synchronized? Default True.
         """
 
         compensation_mode = await self.get_compensation_mode_m2_hexapod()
 
         if compensation_mode.enabled:
             self.log.info(
-                "M2 Hexapod compensation mode enabled. Move will offset with respect to LUT."
+                "M2 Hexapod compensation mode enabled. Move with respect to LUT."
```

Comment:
My suggestion in the `move_camera_hexapod` also applies here, in case it is valid. An `else` statement printing out what happens when the compensation mode is disabled.